apps/account/test/support/factory.ex

```diff
@@ -9,7 +9,7 @@ defmodule Helix.Account.Factory do
   alias Helix.Account.Model.Setting
 
   def account_factory do
-    pk = PK.generate([0x0000, 0x0000, 0x0000])
+    pk = PK.pk_for(Account)
```

Comment:
You actually don't need the PK here since it will be autogenerated when the account is inserted in the database (and thus is not necessary at this point)